### DISCUSSION TITLE
-N switch

### DISCUSSION THREAD

**Discussion by 25bobby** [microsoft/go-sqlcmd#55] [Q&A]
When using -N switch, I run into below error, and also since the `GODEBUG=x509ignoreCN=0 flag will be removed in Go 1.17`, is there anyway that we can skip the tls verify? `tls=skip-verify` Thanks.

```x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0)``` 

```
The GODEBUG=x509ignoreCN=0 flag will be removed in Go 1.17. It enables the legacy behavior of treating the CommonName field on X.509 certificates as a host name when no Subject Alternative Names are present.
```

**Comment by shueybubbles** [comment]
Do you know if this something that should be managed in the go-mssqldb driver based on a connection string setting, or does it already support what you are asking for and go-sqlcmd needs to pass some value?
